package.json

```diff
@@ -128,55 +128,54 @@
     "nunjucks": "^3.2.3",
     "passport": "^0.6.0",
     "passport-local": "^1.0.0",
-    "passport-oauth2": "^1.6.1",
-    "pg": "^8.8.0",
+    "passport-oauth2": "^1.7.0",
+    "pg": "^8.11.0",
     "pg-format": "^1.0.4",
     "querystring": "^0.2.1",
-    "ramda": "^0.28.0",
-    "redis": "^4.5.1",
-    "request": "^2.88.2",
+    "ramda": "^0.29.0",
+    "redis": "^4.6.6",
     "sass-middleware": "^0.0.3",
     "superagent": "^8.0.9",
     "uuid": "^9.0.0"
   },
   "devDependencies": {
     "@types/bunyan": "^1.8.8",
-    "@types/express": "^4.17.16",
+    "@types/express": "^4.17.17",
     "@types/hapi__joi": "^17.1.9",
     "@types/http-errors": "^2.0.1",
-    "@types/jest": "^29.4.0",
-    "@types/node": "^18.11.18",
-    "@types/nunjucks": "^3.2.1",
+    "@types/jest": "^29.5.1",
+    "@types/node": "^20.1.5",
```

Comment:
this should stay at 18 whilst we're on node 18

Reply:
Reverted to version 18.16.12